File: Airline_Web_API/Airline_Web_API/Models/Ticket.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Airline_Web_API.Models
{
    public class Ticket
    {
        public int Id { get; set;}
        public int FlightId { get; set; }
        public int TicketTypeId { get; set; }
        public decimal Price { get; set; }
        public int TicketsLeftNumber { get; set; }
        public string Status { get; set; }

        public Flight Flight { get; set; }
        public TicketType TicketType { get; set; }
    }
}

File: ClientApp/src/app/Models/Ticket.ts
import { TicketType } from './TicketType';
import { Flight } from './Flight';

export interface Ticket {
  id: number;
  ticketType: TicketType;
  flightId: number;
  price: number;
  ticketsLeftNumber: number;
  status: string;
  flight: Flight;
}
